recipes/jest-to-node-test-runner/src/workflow.test.ts

```diff
@@ -0,0 +1,20 @@
+import { resolve } from "node:path";
+import { execPath } from "node:process";
+import { readFile } from "node:fs/promises";
+import { describe, it, type TestContext } from "node:test";
+import { fileURLToPath } from "node:url";
+
+import { spawnPromisified } from "../test/spawn-promisified.ts";
+import { workflow } from "./workflow.ts";
+
+describe("workflow", () => {
+	it("should update bad specifiers and ignore good ones", async (t: TestContext) => {
+		const e2eFixtPath = fileURLToPath(import.meta.resolve("./fixtures/e2e/"));
+		const source = await readFile(resolve(e2eFixtPath, "test.ts"), { encoding: "utf-8" });
+		const result = await workflow(source);
+		const importStatement =
+			'import { describe, it } from "node:test";\nimport { expect } from "expect";\n';
+
+		t.assert.deepStrictEqual(result, `${importStatement}${source.split("\n").slice(1).join("\n")}`);
```

Comment:
we can use snapshot https://nodejs.org/docs/v22.15.0/api/test.html#snapshot-testing

Reply:
Yeah, I will use that in the end, but since I'm changing this a lot I didn't want to have to update them all the time yet